public/jsx/components/card-list.jsx

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var $ = require('jquery');
+var React = require('react');
+var Navigation = require('react-router').Navigation;
+
+var CardItem = require('components/card-item');
+var SubmitButton = require('components/submit-button');
+
+var gettext = require('utils').gettext;
+
+
+module.exports = React.createClass({
+  displayName: 'CardList',
```

Comment:
Yep, that's a plan - I'll go for CardChoice as that's a bit more accurate for a set of radio buttons.